### PR TITLE
Make atlas ids uppercase

### PR DESCRIPTION
**Pull request by onursumer**
Related to #133

### CODE REVIEW

File: components/BiospecimenTable.tsx [hunk 1]
```diff
@@ -21,6 +21,7 @@ export const BiospecimenTable: React.FunctionComponent<IBiospecimenTableProps> =
         {
             name: 'Atlas ID',
             selector: 'atlasid',
+            format: (sample: Entity) => sample.atlasid.toUpperCase(),
```
Comment (onursumer) [inline components/BiospecimenTable.tsx:24]: Should we fix this when processing the data instead?

Comment (inodb) [reply components/BiospecimenTable.tsx:24]: good point - i can prolly tweak it in the python script

Comment (adamabeshouse) [reply components/BiospecimenTable.tsx:24]: +1 - it's been confusing for me to be dealing with casing in the frontend

**Comment by alisman** [comment]
@onursumer @inodb shall we close this?

File: components/BiospecimenTable.tsx (hunk 1 above)
Comment (inodb) [reply components/BiospecimenTable.tsx:24]: fixed it here: https://github.com/ncihtan/htan-portal/pull/118

Comment (onursumer) [reply components/BiospecimenTable.tsx:24]: Should we just close this PR without merging?

**Pull request closed without merging**